docs/bucket/notifications/README.md

```diff
@@ -1,54 +1,71 @@
 # Minio Bucket Notification Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
 
-Events occurring on objects in a bucket can be monitored using bucket event notifications. Event types supported by Minio server are
+This topic describes the bucket notification events and publishing targets supported by Minio Server.
 
-| Supported Event Types | | |
-|:---------------------------|--------------------------------------------|-------------------------|
-| `s3:ObjectCreated:Put`     | `s3:ObjectCreated:CompleteMultipartUpload` | `s3:ObjectAccessed:Head`|
-| `s3:ObjectCreated:Post`    | `s3:ObjectRemoved:Delete`                  |
-| `s3:ObjectCreated:Copy`    | `s3:ObjectAccessed:Get`                    |
+1. [Bucket Notification Event Support](#event-support) 
+2. [Publish Minio Events via AMQP](#publish-minio-events-via-amqp) 
+3. [Publish Minio Events via MQTT](#publish-minio-events-via-mqtt) 
+4. [Publish Minio Events via Elasticsearch](#publish-minio-events-via-elasticsearch) 
+5. [Publish Minio Events via Redis](#publish-minio-events-via-redis) 
+6. [Publish Minio Events via NATS](#publish-minio-events-via-nats) 
+7. [Publish Minio Events via PostgreSQL](#publish-minio-events-via-postgresql) 
+8. [Publish Minio Events via MySQL](#publish-minio-events-via-mysql) 
+9. [Publish Minio Events via Kafka](#publish-minio-events-via-kafka) 
+10. [Publish Minio Events via Webhooks](#publish-minio-events-via-webhooks)
 
```

Comment:
I guess 1.. 10 naming convention is a bit weird and looks odd to the eye. Would it be possible to simply just leave the numbering and just keep it like 

```
- Topic
- Topic
```
![screenshot from 2018-11-27 15-28-29](https://user-images.githubusercontent.com/622699/49118480-37fae200-f259-11e8-863e-eb00704558d2.png)

Other markdown examples which use the `-` style https://github.com/avelino/awesome-go#contents